python/graphstorm/gconstruct/remap_result.py

```diff
@@ -172,6 +253,152 @@ def _get_file_range(num_files, rank, world_size):
 
     return start, end
 
+def remap_node_emb(emb_ntypes, emb_lens, node_emb_dir,
```

Comment:
it seems you have separate implementations for remapping node embeddings and node predictions. why are they different?